src/createMigrationScript/command.js

```diff
@@ -52,11 +52,27 @@ module.exports = async (client) => {
 }
 `.trim();
 
+const getTemplate = templatePath =>
```

Comment:
I don't get it why you're using an an async function returning a promise if you're only using *Sync methods :)

Reply:
Agreed. Initial implementation was using promise rejection to handle errors but after checking the rest of the code style I made it inline.
Will clean up